client/logs/bdqmmm3oo3suave89mrg.log

```diff
@@ -0,0 +1,4 @@
+client.go
```

Comment:
I don't think these files supposed to be under version control. Missing a `.gitignore` directive?